src/main/java/ui/AvatarUi.java

```diff
@@ -3,31 +3,27 @@
-            "▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓▓▓▓▓▓▓▓▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒\n" + 
-            "░░░░░░░░░░░░░░░░▒▒▓███▓▓▓▓▓▓▓█▓▓▒░░░░░░░░░░░░░░░░░\n" + 
-            "░░░░░░░░░░░▒▒▒▒▒▓███▓▓▓▓▓▓███▓▓▓█▒▒▒▒▒▒▒░░░░░░░░░░\n" + 
-            "▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒█████▓▓▓█▓▓▓▓▓▓██▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒\n" + 
-            "░░░░░░░░░░░░░░░░███▓▒▒▒▒▒▒▒▒▒▒▓██▒░░░░░░░░░░░▒░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░███▓▒▒▒▒▒▒▒▒▒▒▓██░░░░░░░░░░░░░░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░▒█▓▓███▓▒▒▓██▓▓██░░░░░░░░░░░░░░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░▓█▓▒▒▒▒▒▒▒▒▒▒▒▒█▓░░░░░░░░░░░░░░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░░▓▓▓▒▒▒▒▒▒▒▒▒▒▓▓▒░░░░░░░░░░░░▒░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░░░▒▓▒▒▒▒▓▓▒▒▒▓▓░░░░░░░░░░░░░░▒░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░░░░▓▓▒▒▓▓▓▓▒▓▓▒░░░░░░░░░░░░░░▒░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░░░░▓▓▓▒▒▒▒▒▒▓▓░░░░░░░░░░░░░░░▒░░▒░\n" + 
-            "░░░░░░░░░░░░░░░░░░▒▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░▒░░▒░\n" + 
-            "░▒░░░░░▒░░░░░░░░▒▓█▒▒▒▓▓▓▓▓▒▒▒▒█▓▒░░░░░░░░░░░▒░░▒░\n" + 
-            "░▒░░▒▓▓▓▒░░░▒▒▓███▓▒░░░▒▓▓░░░▒▒████▓▒▒░░░░░░░▒░░▒░\n" + 
-            "▓▓▒░▒▓▓▓▓▓▓██████▓▓▒░░▒████░░░▒█▓██████▓▓▓▓▓▒▒▒▒▒▒\n" + 
-            "▓▓▓▓██████▓▓▓██▓▓▓▓░░░░▓██░░░░▒█▓▓▓█▓▓▓▓██████▓▓▓▒\n" + 
-            "▓▓████▓▓▓▓▓▓█▓▓▓▓▓█░░░░▓██▒░░░▒█▓▓▓▓▓▓▓▓▓▓▓▓███▓▓▓\n" + 
-            "▒▒▓███▓▓▓▓▓▓▓▓▓▓▓▓█▒░░░███▓░░░▓█▓▓▓▓▓▓▓▓▓▓▓▓▓███▓▓\n" + 
-            "███████▓▓▓▓▓██▓▓▓▓█▓░░░████░░▒██▓▓▓▓█▓▓▓▓▓▓█████▓▓\n" + 
-            "▒▒████████▓▓▓██▓▓▓█▓▓░▒████░░▓████▓██▓▓▓██████████\n" + 
-            "▒▓██████████████████▓▒▒████▒▓▓█████████▓████████▓▓\n" + 
-            "▓▓████████████████████▓████▓████████▓▓▓█████████▓▒\n" + 
-            "▓████████████████████████████████████████████████▒\n" + 
-            "▓████████████████████████████████████████████████▒\n";
+            "                ****#**+                \n" +
+            "              #%%#++**####              \n" +
+            "             %%###%%%##**#*             \n" +
+            "             @%*        *%*             \n" +
+            "             %@+=     =+#@=             \n" +
+            "             #%+*** =***+%=             \n" +
+            "             +#=    =   +#              \n" +
+            "               *=  ++  ++               \n" +
+            "               +*=+++==*                \n" +
+            "               +#*====**                \n" +
+            "              *==*###*= ++              \n" +
+            "           +#%#    +=   =%%*=           \n" +
+            "    *#**#%%%###   *#%=  =###%#%##**++   \n" +
+            "###%%#######**#    %#   =#**########%#**\n" +
+            "++%####**###**%   =##   *#**###***###%++\n" +
+            "##%###########%#  *##=  #############%##\n" +
+            " *%#%%##########+ #%%* *%#########%##%%#\n" +
+            "+%%%%%%####%%#%%#=%%%#=#%%#%%#*#%%%%%%%=\n" +
+            "*@%%%%@%%%#%%%%%%#%%%%##%%%%##%%%@%%%%% \n" +
+            "#@%%%%@%%%%%%%%%%%%@@%%%%%%%%%%%%@%%%%@=\n";
+    
```

Comment:
Sad to see the OG CLI-nton go :( but it is a fix nonetheless.
Have you considered reducing sizes further to help increase screen space - especially on smaller screens?